### DISCUSSION TITLE
Translate Articles

### DISCUSSION THREAD

**Discussion by ghost** [jocmp/capyreader#287] [Feature Requests]
I would love to have a feature to translate articles, but that's a pretty difficult feature.

**Comment by jocmp** [comment]
Yeah it is, unfortunately. This would involve pulling in a 3rd party like Google Translate and/or hosting another service. The former would be expensive and non-free which wouldn't work for F-Droid builds, and the latter would require time and money.

**Reply by jocmp** [reply]
That said if there are options out there, I'm happy to at least investigate them.

**Reply by ghost** [reply]
Unfortunately, no RSS reader does something like this.

**Reply by goliv04053** [reply]
> That said if there are options out there, I'm happy to at least investigate them.

I think there is one.
https://codeberg.org/ManeraKai/simplytranslate

There is an news agregattor called Raven who uses that option.
https://github.com/ksh-b/raven

@jocmp
